ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -239,33 +273,30 @@ Once the well-known endpoint for **issuer server** configuration is resolved, th
           "text_color": "#FFFFFF"
         }
       ],
-      "credential_definition": {
-        "type": "VerifiablePortableDocumentA1",
-        "claims": {
```

Comment:
Second, this `claims` parameter is only available in the HAIP profile but not on draft 13. So, when following the draft 13, we must also add the parameter `vct`.

Reply:
It seems that HAIP and draft 13 are not aligned on this point, which creates this inconsistency.

Reply:
Lets discuss this. @georgepadayatti Can you please elaborate and prepare a short call to settle this?